parser.py

```diff
@@ -52,7 +74,7 @@ def parse(self, content, root=None):
         If the root is not given, new root will created and returned.
         """
         if root is None:
-            root = Element(None)
+            root = Element(content)
```

Comment:
IIUC, the root is used as a placeholder, why we put content in it?

Reply:
let's leave it atm for the sake of debugging purpose, once our code tend to be stable, we will remove it for saving memory